app/models/user.rb

```diff
@@ -102,8 +102,16 @@ def pending_profile?
     pending_profile.present?
   end
 
+  def gpo_verification_pending_profile?
+    gpo_verification_pending_profile.present?
+  end
+
   def pending_profile
-    profiles.gpo_verification_pending.order(created_at: :desc).first
+    gpo_verification_pending_profile
+  end
```

Comment:
should we remove this method? or just alias it?

Reply:
I was inbetween removing this method or keeping it. I was thinking of keeping it for if we have any other reasons to have a `pending_profile` in the future.